octodns/record.py

```diff
@@ -398,6 +385,61 @@ def _process_value(self, value):
         return value
 
 
+class CaaValue(object):
+    # https://tools.ietf.org/html/rfc6844#page-5
+
+    @classmethod
+    def _validate_value(cls, value):
+        reasons = []
+        try:
+            flags = int(value.get('flags', 0))
+            if flags not in (0, 128):
```

Comment:
How come the limit to 128? [The spec](https://tools.ietf.org/html/rfc6844#section-5.1.1) and [this higher level overview](https://support.dnsimple.com/articles/caa-record/) both list `flags` as ranging from 0-255.

Reply:
All of the providers I specifically tried seemed to restrict the flag to 0 or 128. Tbh, I'm tempted to remove the restrictions/validations from octoDNS and just let the providers do what they do as the implementation seems to be really varied. Some allow anything, which seems to agree with the spec, others only allow 0/128 and the 3 "currently" specified `tag` values. 

The downside to not validating in octoDNS is that dry-runs will look like they're going to work, but fail when you go to `--doit`. 

Since the provider validation is already all over the place that's basically unavoidable, thus my inclination to just let anything through.

Reply:
Makes sense..maybe once the spec is more widely deployed there will be more consistency/sanity checks that can be added. So..yeah..maybe it makes sense to just limit it to 0-255 (i.e. basically no validation other than it will fit in a byte as is required for the record response format specified [in the rfc](https://tools.ietf.org/html/rfc6844#section-5.1).).